app/views/dashboard/index.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Homepage</h1>
+
+
+<!-- link to creating new notes and photos  -->
+<!-- display all of the notes and photos the user made in a list -->
+<div class="row">
+  <div class="col s6">
+    <%= link_to "Create a Photo", new_photo_path, class: "btn" %>
+  </div>
+  <div class="col s6">
+    <%= link_to "Create a Note", new_note_path, class: "btn" %>
```

Comment:
How do these look on a medium or larger screen? I feel like there is going to be a large space between them